tests/foreman/cli/test_host.py

```diff
@@ -1572,6 +1573,8 @@ def test_positive_provision_baremetal_with_uefi_secureboot():
 @pytest.fixture(scope="function")
 def setup_custom_repo(target_sat, module_org, katello_host_tools_host):
     """Create custom repository content"""
+    # disable SCA for the org
+    target_sat.api.Organization(id=module_org.id, simple_content_access=False).update(['simple_content_access'])
```

Comment:
can you try `module_org.sca_disable()` here instead of an update?

Reply:
so this actually doesn't work:
```
'{"displayMessage":"Current organization does not have a manifest imported.","errors":["Current organization does not have a manifest imported."]}'
```
but updating the attribute directly does

Reply:
I filed a BZ:
https://bugzilla.redhat.com/show_bug.cgi?id=2159974

to workaround this, I decided to override a `module_org` fixture by a custom one, that creates the org with sca set to false.

Reply:
the bz is now resolved, i'm going to use originally proposed `module_org.sca_disable()` with some logic around it (finalizer, etc.)